File: src/tp1/Estoque.java
package tp1;

public class Estoque {
	private int quantidade;
	private float preco;
	private Montaria montaria;

	public Estoque(Montaria montaria, int quantidade, float preco){
		this.quantidade = quantidade;
		this.preco = preco;
		this.montaria = montaria;
	}
	
	public Montaria getMontaria() {
		return montaria;
	}

	public void setMontaria(Montaria montaria) {
		this.montaria = montaria;
	}

	public int getQuantidade() {
		return quantidade;
	}

	public void setQuantidade(int quantidade) {
		this.quantidade = quantidade;
	}

	public float getPreco() {
		return preco;
	}

	public void setPreco(float preco) {
		this.preco = preco;
	}

}

File: python/Estoque.py
from Entidade import Entidade

class Estoque (Entidade):

    def __init__(self, quantidade, preco, montaria):
        self.__quantidade = quantidade
        self.__preco = preco
        self.__montaria = montaria

    def getMontaria(self):
        return self.__montaria

    def setMontaria(self, montaria):
        self.__montaria = montaria

    def getQuantidade(self):
        return self.__quantidade

    def setQuantidade(self, quantidade):
        self.__quantidade = quantidade

    def getPreco(self):
        return self.__preco

    def setPreco(self, preco):
        self.__preco = preco


